pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentStickyKeyDispatcherMultipleConsumers.java

```diff
@@ -322,15 +322,9 @@ protected synchronized boolean trySendMessagesToConsumers(ReadType readType, Lis
             }
             // readMoreEntries should run regardless whether or not stuck is caused by
             // stuckConsumers for avoid stopping dispatch.
-            sendInProgress = false;
             topic.getBrokerService().executor().execute(safeRun(this::readMoreEntries));
```

Comment:
Should we remove this tow line(line-325, line-327) and make this method return `true`? Then `readMoreEntries` will be triggered by `super.readEntriesComplete`.

Reply:
You are right.